x-pack/plugins/ml/public/application/components/data_grid/data_grid.scss

```
@@ -0,0 +1,4 @@
+div.mlDataGrid .euiDataGridRowCell--boolean {
+  text-transform: none;
+}
+
```

```suggestion
.mlDataGrid {
  .euiDataGridRowCell--boolean {
     text-transform: none;
  }
}

```
We usually use nesting whenever possible in Sass.

---

Good suggestion - updated in ca000bc97d7dd455963543c7918c5d32e04d9ae2